src/main/java/slimeknights/mantle/client/book/HTMLUtils.java

```diff
@@ -0,0 +1,89 @@
+package slimeknights.mantle.client.book;
+
+
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.network.chat.Style;
+import net.minecraft.network.chat.TextColor;
+
+import javax.annotation.Nullable;
+import java.util.stream.Collectors;
+
+public class HTMLUtils {
```

Comment:
This whole class needs a lot of javadocs. For any new method/class, it should either have `@Override` or a javadoc.

I do want to one day put javadoc exports on our website as an aside if you are interested in figuring that out.

Reply:
I said ***either*** `@Override` or a javadoc. Its not inheriting anything so it doesn't have `@Override`, thus it should have a javadoc.

My point is don't add javadocs for things that already override unless it adds important clarifying info.